Source/Blazorise.Tailwind/TailwindClassProvider.cs

```diff
@@ -593,9 +593,19 @@ public override string DropdownFixedHeaderVisible( bool visible )
 
     public override string DropdownMenuRight() => "b-dropdown-menu-right";
 
-    public override string DropdownToggle( bool isDropdownSubmenu, bool outline ) => isDropdownSubmenu
-        ? "b-dropdown-toggle-submenu block flex flex-row justify-between w-full py-2 px-4 hover:bg-gray-100 dark:hover:bg-gray-600 dark:hover:text-white"
-        : "b-button b-dropdown-toggle focus:ring-4 focus:outline-none font-medium text-sm text-center inline-flex items-center";
+    public override string DropdownToggle( bool isDropdownSubmenu, bool outline )
+    {
+        string baseClass = isDropdownSubmenu
+            ? "b-dropdown-toggle-submenu block flex flex-row justify-between w-full py-2 px-4 hover:bg-gray-100 dark:hover:bg-gray-600 dark:hover:text-white"
+            : "b-button b-dropdown-toggle focus:outline-none font-medium text-sm text-center inline-flex items-center";
+
+        if ( outline )
+        {
+            baseClass += " focus:ring-4";
```

Comment:
Every time you need to combine string you must use StringBuilder instead of concatenation with `+=`. The `+=` is much slower.